mipengine/common/sql_injection_guard.py

```diff
@@ -0,0 +1,42 @@
+import re
+from functools import wraps
+
+
+def sql_injection_guard(func):
+    wraps(func)
+
+    def wrapper(*args, **kwargs):
+        all_args = list(args) + list(kwargs.values())
+        for arg in all_args:
+            if type(arg) == str:
+                if (
+                    not arg.isidentifier()
+                    and not arg.isalnum()
+                    and not has_proper_db_location_format(arg)
+                ):
+                    raise ValueError(f"Not allowed character in argument: {arg}")
+            elif type(arg) == list:
+                for item in arg:
+                    if (
+                        not item.isidentifier()
+                        and not item.isalnum()
+                        and not has_proper_db_location_format(item)
+                    ):
+                        raise ValueError(f"Not allowed character in argument: {item}")
+        return func(*args, **kwargs)
+
+    return wrapper
+
+
+def has_proper_db_location_format(db_location: str):
```

Comment:
Use the standard term `db_socket_address` instead of `db_location`, see [here](https://en.wikipedia.org/wiki/Network_socket)